src/queue/Queue.hxx

```diff
@@ -85,6 +85,9 @@ struct Queue {
 	/** play back songs in random order? */
 	bool random = false;
 
+	/** Last loaded playlist */
+	std::string last_loaded_playlist = "";
```

Comment:
This initializer has no effect (except for adding runtime overhead)

Reply:
Removed